src/DashboardService.Models.Dto/Requests/Group/PatchGroupRequest.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+
+namespace LT.DigitalOffice.DashboardService.Models.Dto.Requests.Group;
+
+public record PatchGroupRequest
+{
+  public Guid BoardId { get; set; }
+  public string GroupName { get; set; }
+  public bool GroupIsActive { get; set; }
+  public Guid? GroupModifiedBy { get; set; }
```

Comment:
unnecessary. you can get it yourself. no need to get it from frontend. GroupModifiedAtUtc also

Reply:
++

Reply:
++ what?